### ISSUE TITLE
Unofficial diffuser support

### ISSUE THREAD

**Issue by KaustubhPatange** [layerdiffusion/LayerDiffusion#27]
Appreciated your work on transparent image generation. I've been working on an unofficial implementation of LayerDiffuse mostly forked from forge's extension to my repo: https://github.com/KaustubhPatange/Diffuser-layerdiffuse.

However I've noticed many flaky outputs where the transparent (intermediate) image sometimes does not get generated and has a weird output. I believe it solely depends on the model you use for example choosing Juggernaut XL over Realistic Vision XL tends to provide much cleaner outputs.

**Comment by KaustubhPatange** [comment]
Yeah, this is more of a model issue. Trying with different seed would eventually give you a transparent tile image. But in VAE decoding it may or may not produce transparent images.

**Comment by G-force78** [comment]
It has huge VRAM requirements why is that?

**Comment by zhanpengxin** [comment]
你好，我已收到你的邮件，我会及时处理^_^
